import_export/export-wordpress.py

```diff
@@ -0,0 +1,95 @@
+   },
+   "id" : 24,
+   "link" : "https:\/\/ctoassetsstg.wpengine.com\/2020\/10\/01\/second-post\/",
+   "modified_gmt" : "2020-10-01T11:15:22",
+   "wptitle" : {
+      "rendered" : "Segunda publicación"
+   }
+}
+"""
+
+
+SOURCE_PATH = 'source_files/en'
+GIT_BRANCH = f"{os.environ.get('DEVELOPER_USERNAME')}/{os.environ.get('ENV')}/local"
+ROOT_PATH = os.path.abspath(os.path.join(os.path.dirname( __file__ ), '../../tms-data'))
+GIT_REPO_PATH = f'{ROOT_PATH}/.git'
+COMMIT_MESSAGE = 'Update shared repository'
+
+SOURCE_DIRECTORY = 'source_files'
+SOURCE_LANGUAGE = 'en'
+
+def export_wordpress():
+
+  # Get the latest commit?
+  repo = Repo(GIT_REPO_PATH)
+  commits_list = list(repo.iter_commits())
+
+  a_commit = commits_list[0]
+  b_commit = commits_list[-1]
+
+  diffs = a_commit.diff(b_commit)
+  files = [f.a_path for f in diffs]
+
+  for f in files:
+    if f.startswith(f"{SOURCE_DIRECTORY}"):
+      if not f.startswith(f"{SOURCE_PATH}"):
+        target = f"{ROOT_PATH}/{f}"
+        basename = os.path.basename(target)
+        if basename.startswith("wp"):
+          # print(target)
+          with open(target, "rb") as json_file:
+            filename, file_extension = os.path.splitext(basename)
+
+            data = json.load(json_file)
+            print(data)
+            print(file_extension)
+            id = data["id"]
+            title = data["wptitle"]["rendered"]
+            content = data["content"]["rendered"]
+            lang =  file_extension[1:]
+            
+            update_wordpress(id, lang, title, content)
+
+def update_wordpress(id, language, title, content, excerpt=""):
+  export_url = os.environ.get('WP_EXPORT_URL')
+  message = {
+      "content": content,
+      "title": title,
+      "excerpt": excerpt,
+      "status": "published"
+  }
+  url = f"{export_url}/wp-json/elsa/v1/translate/{id}/{language}"
+  user = os.environ.get('WP_EXPORT_USER')
+  password = os.environ.get('WP_EXPORT_PASSWORD')
+  credentials = user + ':' + password
+  token = base64.b64encode(credentials.encode())
+  header = {'Authorization': 'Basic ' + token.decode('utf-8')}
+
+  response = requests.post(url, headers=header, json=message)
```

Comment:
Aditya - If the request failed (eg because of bad credentials), what would be the best way to surface the error message?